<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_operators">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Operators</string>
    </property>
    <widget class="QTabWidget" name="tabWidget_operators">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>291</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Artihmetic</string>
      </attribute>
      <widget class="QRadioButton" name="radioButton_add">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_substract">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>81</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>substract</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_multiply">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>71</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Multiply</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_divide">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>61</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Divide</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Algebraic</string>
      </attribute>
      <widget class="QRadioButton" name="radioButton_comp">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>100</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Composition</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_subs">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>100</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Substitution</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Trigonometric</string>
      </attribute>
      <widget class="QRadioButton" name="radioButton_cos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>51</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_sin">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>51</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_tan">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>88</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>tan</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_appfunc">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>591</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Application of functions</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_func">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>451</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>4</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>104</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>24</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string extracomment="Direction of Application"/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/arrow.svg</normalon>
       </iconset>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>g(y1,y2)</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>h(y1)</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>n</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>f(x1,x2)</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>k(x1)</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>m</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_delete_row">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Delete row</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+R</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_delete_column">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>Delete column</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_delete_cell">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete cell</string>
     </property>
     <property name="shortcut">
      <string>Del</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_create">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>180</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Func/Const</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2d">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>2D view of cell</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_interResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>761</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Intermediate results</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_inter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>621</width>
       <height>192</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="rowCount">
      <number>6</number>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QPushButton" name="pushButton_del_inter_cell">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete cell</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_inter_cell">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear all</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_export_exp">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>113</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Export expression</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Software"/>
    <addaction name="action_User_Manual"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_New">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/filenew.png</normaloff>:/filenew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create a grid data file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/fileopen.png</normaloff>:/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/filesave.png</normaloff>:/filesave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/filequit.png</normaloff>:/filequit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout_Software">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/editzoom.png</normaloff>:/editzoom.png</iconset>
   </property>
   <property name="text">
    <string>About Software</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/filesaveas.png</normaloff>:/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>&amp;SaveAs</string>
   </property>
  </action>
  <action name="action_User_Manual">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/editzoom.png</normaloff>:/editzoom.png</iconset>
   </property>
   <property name="text">
    <string>&amp;User Manual</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/undo.svg</normaloff>:/undo.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/redo.svg</normaloff>:/redo.svg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_clear_inter_cell</sender>
   <signal>clicked()</signal>
   <receiver>tableWidget_inter</receiver>
   <slot>clearContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>705</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
